redhat/securitydataapi/redhat.go

```diff
@@ -33,6 +34,12 @@ func Update() error {
 }
 
 func update(year int) error {
+	dirPath := filepath.Join(utils.VulnListDir(), apiDir)
+	log.Printf("Remove RedHat API directory %s", dirPath)
+	if err := os.RemoveAll(dirPath); err != nil {
+		return xerrors.Errorf("failed to remove RedHat API directory: %w", err)
+	}
+
```

Comment:
We remove the `api` directory for each year, so after adding a year we remove previous years.

Let's move this to `Update()`  function

Reply:
Thank you for your review, I have fixed it as follows:
https://github.com/aquasecurity/vuln-list-update/compare/983a7497e431768e82fbce6a8c951899301a8895..938543a44e2bdbcceaca2ecd5eea92280bd7164c